src/vector/jitsi/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Jitsi Widget</title>
+</head>
+<body>
+<div id="jitsiContainer"><!-- the js will put the conference here --></div>
+<div id="joinButtonContainer">
+    <div class="join-conference-boat">
```

Comment:
What does "boat" mean here?

Reply:
It means I copied too much from Dimension and didn't review it for code quality ._.

Likewise for the camel case issues.

"Boat" is a super technical term for "floats in the roughly the middle of the container" btw.